ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -382,6 +398,22 @@ router bgp {{ router_bgp.as }}
 {%             endif %}
    {{ maximum_routes_cli }}
 {%         endif %}
+{%         if neighbor.missing_policy_af_all is arista.avd.defined %}
+{%             for direction in ['in', 'out'] %}
+{%                 set dir = 'direction_' ~ direction %}
+{%                 set action = neighbor.missing_policy_af_all[dir].action %}
+{%                 if action is arista.avd.defined %}
+{%                     set missing_policy_cli = "neighbor " ~ neighbor.ip_address ~ " missing-policy address-family all" %}
+{%                     if neighbor.missing_policy_af_all[dir].include is arista.avd.defined %}
+{%                         set include_list = neighbor.missing_policy_af_all[dir].include | arista.avd.natural_sort %}
```

Comment:
I am not sure we should natural sort here. The order should match the order on EOS. I am thinking it will be simpler with a boolean per include instead of a list.